static/js/textbox.html

```diff
@@ -0,0 +1,113 @@
+<html>
+<head>
+
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.7.1/jquery.min.js" type="text/javascript"></script>
+    <script src="https://edx-stanford-dev.raccoongang.com/course-content/2017_N/static/js/server_interaction.js" type="text/javascript"></script>
+    <!-- <script src="http://52.210.53.36:8080/static/server_interaction.js" type="text/javascript"></script> -->
+    <script type="text/javascript">
+        function getUrlVars() {
+            var vars = {};
+            var parts = window.location.href.replace(/[?&]+([^=&]+)=([^&]*)/gi, function(m,key,value) {
+                vars[key] = value;
+            });
+            return vars;
+        }
+
+        vars = getUrlVars();
+        showLink = (vars['showlink'] == 'true' || vars['showlink'] == 'True' || vars['showlink'] == 'yes');
+        showComplete = (vars['completed'] == 'true' || vars['completed'] == 'True' || vars['completed'] == 'yes');
+        divname = vars['iframe'];
+
+        function resizeIframe() {
+          height = document.getElementById('all').offsetHeight + 30;
+          window.parent.document.getElementById(divname).height = height + 'px';
+        }
+
+        function display_problem_callback(e)
+        {
+            var text_div = document.getElementById('text');
+            if(e.detail.display == true) {
+                //We're in the right place, display the problem
+                text_div.innerHTML = window.parent.document.getElementById(vars['div']).getAttribute('text');
+            } else {
+                //We're in the wrong place, redirect to correct page
+                window.top.location.href = e.detail.redirect
+            }
+            resizeIframe();
+        }
+
+        function display_problem_error_callback(e)
+        {
+            var text_div = document.getElementById('text');
+            text_div.innerHTML = "Error connecting to RL server.</br> Probably you are not enrolled in Adapt application, if it is your case visit Adapt enrolling page for that section, usually 'Introduction' or connect to the support team if automatic enrollment is not provided.</br> Details: " + e.detail.err + "</br>" + e.detail.textStatus;
```

Comment:
It looks like nobody knows what `RL` stands for. So I guess we should change it to `EdxAdapt` here and everywhere to at least not confuse ourselves.
Not in this PR though.